sip-ri/src/test/java/gov/nist/javax/sip/parser/extensions/TargetDialogParserTest.java

```diff
@@ -7,13 +7,20 @@ public class TargetDialogParserTest extends ParserTestCase {
     @Override
     public void testParser() {
         String[] targetDialogHeaders = {
-                "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\n",
-                "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\n",
-                "Target-dialog: 1@10.0.0.1\n",
-                "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f\n",
-                "Target-dialog: 281794\n"
+                "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\r\n",
+                "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\r\n",
+                "Target-dialog: 1@10.0.0.1\r\n",
+                "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f\r\n",
+                "Target-dialog: 281794\r\n",
+        		//local tag
+                "Target-dialog: f0b40bcc-3485-49e7-ad1a-f1dfad2e39c9@10.5.0.53\r\n",
+                //remote tag 
+                "Target-dialog: f81d4fae-7dec-11d0-a765-00a0c91e6bf6@foo.bar.com\r\n",
+                // local tag 
+                "Target-dialog: kl24ahsd546folnyt2vbak9sad98u23naodiunzds09a3bqw0sdfbsk34poouymnae0043nsed09mfkvc74bd0cuwnms05dknw87hjpobd76f@example.com\r\n" 
```

Comment:
This still doesn't include tests for local and remote tag. See https://www.rfc-editor.org/rfc/rfc4538#section-10

"Target-Dialog: fa77as7dad8-sd98ajzz@host.example.com;local-tag=kkaz-;remote-tag=6544"

Reply:
My apologies, I made TargetDialogParser to likely extract this identifier during parsing, but not to explicitly separate as a tag within the class structure. Thank you for the example